packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +315,57 @@ const Updates = decorate([
             </CardBlock>
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <form id={state.channelsFormId} className='form'>
+                <div className='form-group'>
+                  <Select
+                    autoSelectSingleOption={false} // avoid selecting *unlisted channel* before public channels are retrieved
```

Comment:
I don't think this is still necessary because we only compute the options after having fetched the public channels.

Reply:
:heavy_check_mark: